Applications/VGG/res/vgg.ini

```diff
@@ -0,0 +1,228 @@
+# Network Section : VGG16 - Cifar100
+#
+# conv3_64 - conv3_64, max_pooling, conv3_128 - conv3_128, max_pooling, conv3_256 - conv3_256 - conv3_256, max_pooling,
+# conv3_512 - conv3_512 - conv3_512, max_pooling, conv3_512 - conv3_512 - conv3_512, fc_4096, fc_4096, fc_100, softmax
+#
+[Model]
+Type = NeuralNetwork	# Network Type : Regression, KNN, NeuralNetwork
+Learning_rate = 1e-4 	# Learning Rate
+Epochs = 1500		# Epochs
+Optimizer = adam 	# Optimizer : sgd (stochastic gradien decent),
+ 	    		#             adam (Adamtive Moment Estimation)
+Loss = cross  		# Loss function : mse (mean squared error)
+                        #                       cross ( for cross entropy )
+Save_Path = "model.bin"  	# model path to save / read
+batch_size = 32		# batch size
+beta1 = 0.9 		# beta 1 for adam
+beta2 = 0.999	# beta 2 for adam
+epsilon = 1e-7	# epsilon for adam
+
+# Layer Section : Name
+[inputlayer]
+Type = input
+Input_Shape = 3:32:32
+
+# Layer Section : Name
+[conv2d_c1_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
```

Comment:
this is already done, but for the future, can let the default initializer handle many things like `bias_initializer`, `padding`, `stride` to reduce the ini file size and for readability.